keras_cv/models/resnet_v2_backbone.py

```diff
@@ -369,35 +301,13 @@ def __init__(
         stackwise_blocks,
         stackwise_strides,
         include_rescaling,
-        include_top,
```

Comment:
one thing I don't like is that `stackwise_blocks`, `stackwise_strides`, are now required parameters --- most users don't really care about the details of these arguments but rather want something like ResNet50V2?   Are we basically committing to users only constructing ResNets from pretrained in this change?  I see we've removed ResNet50/etc now

Reply:
Yes, you either use the generic constructor or you use `from_preset`.

Reply:
See below discussion about reintroducing the keras.applications style aliases.